### PR TITLE
Expand the stat width with "flex: 1"

### PR DESCRIPTION
**Pull request by denisw**
This also seems to remove the need for the IE11 fix in the code (works fine in IE).

screen shot 2018-02-27 at 17 32 40

### CODE REVIEW

**Comment by sarunast** [comment]
Deploy preview for *masifunde* ready!

Built with commit c38f7b4209e0eb531cd6fbb9f8cfce9ab212e44e

https://deploy-preview-199--masifunde.netlify.com

File: components/Stat.js [hunk 1]
```diff
@@ -10,10 +10,10 @@ const StatContainer = styled.div`
   display: flex;
   flex-direction: column;
   align-items: center;
-  
-  /* IE11 fix */
-  flex-shrink: 1;
+
+  flex: 1;
```
Comment (erikjalevik) [inline components/Stat.js]: Might be worth to ping @torkjels about this change, which he introduced in this commit only two weeks ago: https://github.com/futurice/masifunde-fe/commit/d217e63ff43785fa55b71ffe82271f366a0594bb

Comment (torkjels) [reply components/Stat.js]: So `flex: 1` is the same as 
```
flex-grow: 1;
flex-shrink: 1;
flex-basis: 0px;
```
( https://css-tricks.com/almanac/properties/f/flex/#article-header-id-5 )

Testing it out a bit now, it seems like the key to that fix was the `flex-basis`, not the `flex-shrink`.

What is this PR aimed at fixing, @denisw ?

Comment (erikjalevik) [reply components/Stat.js]: It fixes those stats column being rendered exceedingly narrowly.

**Pull request merged**